### PR TITLE
Add bug-report command line

### LINKED ISSUE #142: Create bug-report command
Useful information for solving reported bugs:
- kytos-utils
  - version
    - number
    - GitHub (with hash) or pypi install?
  - installed NApps and versions (simple: kytos napps list)
- python-openflow
  - version
    - number
    - GitHub (with hash) or pypi install?
- kytos
  - version
    - number
    - GitHub (with hash) or pypi install?

### PR DESCRIPTION
**Pull request by macartur**
Fix #142

### CODE REVIEW

File: kytos/cli/commands/bug_report/api.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+"""Translate cli commands to non-cli code."""
+import re
+import subprocess
+
+
+class BugReportAPI:
+    """Class to retrieve Kytos environment informations.
+
+    Class is responsible to collect system information, python environment,
```
Comment (diraol) [inline kytos/cli/commands/bug_report/api.py]: `Collect system information, python environment, python packages and installed napps to print this report to the user aiming improve bug reports.`

File: kytos/cli/commands/bug_report/api.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    def bug_report(cls, args):  # pylint: disable=unused-argument
+        """Run all bug report."""
+        cls.system_report()
+        cls.python_environment()
+        cls.python_packages_report()
+        cls.kytos_environment_report()
+
+    @classmethod
+    def system_report(cls):
+        """Display all system informations.
```
Comment (diraol) [inline kytos/cli/commands/bug_report/api.py]: `Display system information.`

File: kytos/cli/commands/bug_report/api.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+        This method shows distribution release and system informations.
```
Comment (diraol) [inline kytos/cli/commands/bug_report/api.py]: `Print distribution release and system information.`

File: CHANGELOG.rst [hunk 4]
```diff
@@ -7,6 +7,8 @@ This is a log of changes made to the *kytos-utils* project.
 ********************************
 Added
 =====
+- Command to help new user with bug report.
```
Comment (diraol) [inline CHANGELOG.rst]: `Command that print information usefull for a bug report`

**Comment by macartur** [comment]
Updated

File: kytos/cli/commands/bug_report/api.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+            if line.startswith('-e '):
+                url, name = line.split('#egg=')
+                url = url.replace('-e ', '')
+                repository, hash_number = cls._parse_github_install(url)
+                print(f'{name:<30} | {repository:<30} | {hash_number:<30}')
+
+    @classmethod
+    def _parse_github_install(cls, url):
+        """Parse the repository url and get the github path and commit hash."""
+        pattern = "(github.com:.*).git@(.{8})"
```
Comment (cemsbr) [inline kytos/cli/commands/bug_report/api.py]: It cannot parse the repo and version of a kytos project installed from a local clone. When so, the freeze output is, for example: `git://github.com/kytos/kytos.git@0f87413fd60de588dc8f821fb00cebfb8489d2ed`

**Pull request merged**